product_research/competitor_cuj.md

```diff
@@ -0,0 +1,152 @@
+# A3 CUJ Runthrough + Demo
+
+**Team PropertyVision**
+
+**Group Members:**
+- Sunny Kim – 100084121931
+- Carol Meng - 1008593675
+- David Barsamyan - 1008131601
+
+**Submission Date:** 2025/09/24
+
+## ⚡ TL;DR (Max 65 words)
+
+We tested multiple property valuation tools as Canadian homeowners seeking accurate home estimates. Started with Masterkey/Zillow but found hit geographical restrictions. Then Zolo which gave an estimate $1million off from similar properties in the area. Finally Wahi, which provided $1.1million estimate. Process took 10 minutes but prices seem low for 3 bedroom unit. No tools used photo data for enhanced accuracy.
+
+## 🎯 Goal
+
+As a **Canadian homeowner considering selling my 3-bedroom house**, I want to **get an accurate current valuation of my property using available online tools**, so that I can **understand my home's market value and determine realistic pricing expectations before making selling decisions**.
```

Comment:
Isn't this too specific? The problem we want to solve is for homeowners in general rather than for homeowners with 3-bedroom houses right?